docs/admission-control.md

```diff
@@ -21,7 +21,7 @@ func main() {
     validatingController := resource.SimpleValidatingAdmissionController{
         ValidateFunc: func(ctx context.Context, request *resource.AdmissionRequest) error {
             // Check that the name is allowed
-            if request.Object.StticMetadata().Name == "not-allowed" {
+            if request.Object.GetStaticMetadata().Name == "not-allowed" {
```

Comment:
Might be useful just to change this completely away from using `StaticMetadata`, as we now have getters for all metadata properties:
```suggestion
            if request.Object.GetName() == "not-allowed" {
```